lib/pages/onboarding/onboarding_carousel/basic_carousel_page.dart

```diff
@@ -0,0 +1,77 @@
+import 'package:app/assets/StyleFrom.dart';
+import 'package:app/viewmodels/onboarding_model.dart';
+import 'package:flutter/material.dart';
+
+class BasicCarouselPage extends StatelessWidget {
```

Comment:
I really like this! Do you think we can take this any further? Doing the First page with this is probably a bit of a strech but what would be nice is if we add:
An argument to include an optional button
And a boolean (default to True) for whether we want the skip page
Adding those 2 in should allow us to also do page 4 with this class